helm/reana/README.md

```diff
@@ -124,6 +124,7 @@ collisions.
 | `kubernetes_jobs_max_user_memory_limit`                                                                   | Maximum custom memory limit that users can assign to their job containers via `kubernetes_memory_limit` hint in `reana.yaml`. Exceeding this limit will terminate the container. Please see the following URL for possible values <https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory>.                                                                                                | None                                                                                                   |
 | `kubernetes_jobs_timeout_limit`                                                                           | Default timeout for user's jobs in _seconds_. Exceeding this time will terminate the job. Please see the following URL for more details <https://kubernetes.io/docs/concepts/workloads/controllers/job/#job-termination-and-cleanup>. Default value: 604800 seconds = 7 days.                                                                                                                                                        | 604800                                                                                                 |
 | `kubernetes_jobs_max_user_timeout_limit`                                                                  | Maximum custom timeout in _seconds_ that users can assign to their jobs. Please see the following URL for more details <https://kubernetes.io/docs/concepts/workloads/controllers/job/#job-termination-and-cleanup>. Default value: 1209600 seconds = 14 days.                                                                                                                                                                       | 1209600                                                                                                |
+| `kueue.enabled`                                                                                           | Enable Kueue integration. If true, the Helm chart will create ClusterQueue and LocalQueue resources.                                                                                                                                                                                                                                                                                                                                 | false                                                                                                  |
```

Comment:
This help text is now outdated. Please say rather than it presupposes having a Kueue installed before, with two local queues named batch-queue and job-queue.  (Which is sort of hard-coded right now; admins cannot customise it yet.)